### PR TITLE
✨ Show warning for non meaningful subjects

### LINKED ISSUE #545: Force meaningful subject
Sometimes, subjects for FOI requests are pretty useless - for example, some people simply write "Anfrage" into the subject line. We should curate a list of subjects that are not permissible (while still allowing the word "Anfrage" in this case, of course).

### PR DESCRIPTION
**Pull request by pajowu**
Closes #545

### CODE REVIEW

File: froide/foirequest/views/make_request.py [hunk 1]
```diff
@@ -133,6 +133,9 @@ def get_js_context(self):
             "settings": {
                 "user_can_hide_web": settings.FROIDE_CONFIG.get("user_can_hide_web"),
                 "user_can_create_batch": self.can_create_batch(),
+                "non_meaningful_subject_regex": settings.FROIDE_CONFIG.get(
```
Comment (stefanw) [inline froide/foirequest/views/make_request.py:136]: This should have a default value of `[]` so the JS `.map` doesn't crash on `null` value.

**Review by stefanw — APPROVED** [review]

**Comment by stefanw** [comment]
Generally, I would leave translation files out of PRs. They are a pain to conflict merge (at least I have not found a good way). I try to add translations in a separate commit on main before deploy.

File: froide/foirequest/views/make_request.py (hunk 1 above)
Comment (pajowu) [reply froide/foirequest/views/make_request.py:136]: ✅

**Comment by pajowu** [comment]
I added the default and removed the translation file

**Pull request merged**